src/java/org/apache/cassandra/db/SortedLocalRanges.java

```diff
@@ -46,17 +46,17 @@
     private static final Logger logger = LoggerFactory.getLogger(SortedLocalRanges.class);
 
     private final StorageService storageService;
-    private final CompactionRealm realm;
+    private final ColumnFamilyStore cfs;
     private final long ringVersion;
     private final List<Splitter.WeightedRange> ranges;
     private final Map<Integer, List<PartitionPosition>> splits;
 
     private volatile boolean valid;
 
-    public SortedLocalRanges(StorageService storageService, CompactionRealm realm, long ringVersion, List<Splitter.WeightedRange> ranges)
+    public SortedLocalRanges(StorageService storageService, ColumnFamilyStore cfs, long ringVersion, List<Splitter.WeightedRange> ranges)
```

Comment:
`storageService` is no longer used

Reply:
With my new commit, which fixes `realm` back, it is used again.